controller/emailVerify.js

```diff
@@ -0,0 +1,21 @@
+var exports =module.exports={}
```

Comment:
Use just ES6 or ES5 syntax, dont mix

use Let for variables and const

Reply:
ok